gdal.go

```diff
@@ -1849,3 +1847,29 @@ func VSIFReadL(nSize, nCount int, file VSILFILE) []byte {
 
 	return data
 }
+
+// Fetch buffer underlying memory file.
+func VSIGetMemFileBuffer(filename string, unlinkAndSeize bool) ([]byte, error) {
```

Comment:
Just wondering what is VSI?

Reply:
I _think_ VSI refers to GDAL's virtual file system (or has something to do with it). I'm having trouble finding info about it.